src/app/docs/getting-started/configuring-providers/page.mdx

```diff
@@ -160,4 +160,48 @@ spec:
   provider: Ollama
   ollama:
     host: http://ollama.ollama.svc.cluster.local
-```
\ No newline at end of file
+```
+
+## Configuring Custom Models
+
+Often you may want to use a custom model with an existing provider, for example if you're using a self-hosted model exposed via the OpenAI API.
+Or if you're using a proxy to the OpenAI API.
+
+Autogen requires certain information about any model to be able to use it. Autogen ships with a default set of models, but you can also add custom models.
```

Comment:
does the reader care about Autogen here? Perhaps just say "We require certain inf..."

Reply:
```suggestion
We require certain information about any model to be able to use it. We ship with a default set of models, but you can also add custom models.
```